ui/app/css/itcss/components/sections.scss

```diff
@@ -403,10 +403,10 @@ textarea.twelve-word-phrase {
 }
 
 .qr-ellip-address, .ellip-address {
-  /*rtl:ignore*/
```

Comment:
Why was this removed? I think this is still required to prevent the direction of the address from being flipped when using a RTL locale

Reply:
Is... is this comment used? 🤦

Reply:
Yes - the `rtl` comments are directives for `rtlcss`, which we use via `gulp-rtlcss`

Reply:
I've added it back, that's unfortunate